packages/nc-gui/components/dashboard/settings/data-sources/CreateBase.vue

```diff
@@ -383,6 +392,43 @@ watch(
         <a-input v-model:value="(formState.dataSource.connection as SQLiteConnection).connection.filename" />
       </a-form-item>
 
+      <template v-else-if="formState.dataSource.client === ClientType.SNOWFLAKE">
+        <!-- Account -->
+        <a-form-item label="Account" v-bind="validateInfos['dataSource.connection.account']">
+          <a-input v-model:value="formState.dataSource.connection.account" class="nc-extdb-account" />
+        </a-form-item>
+
+        <!-- Username -->
+        <a-form-item :label="$t('labels.username')" v-bind="validateInfos['dataSource.connection.user']">
+          <a-input v-model:value="formState.dataSource.connection.username" class="nc-extdb-host-user" />
```

Comment:
The validation rule name has to be matched with the property name. Currently, there is no validation on userame.

![image](https://user-images.githubusercontent.com/35857179/209057395-f5d743f6-570c-40dc-b515-3d23ffbd096c.png)

In L403, please change to

```
v-model:value="formState.dataSource.connection.user"
```